keyboards/niu_mini/keymaps/yttyx/keymap.c

```diff
@@ -0,0 +1,145 @@
+
+#include QMK_KEYBOARD_H
+
+enum layers {
+  BA,           // Base        (Balance Twelve mirror varian)
+  P1,           // Punctuation (1)
+  P2,           // Punctuation (2)
+  P3,           // Punctuation (2)
+  NC,           // Numerals / Cursor control
+  FV            // Function keys / Cursor control (Vim)
+};
+
+// Abbreviations - general
+#define xxxxxxx         KC_NO
+#define _______         KC_TRNS
```

Comment:
This is already included in `#include QMK_KEYBOARD_H`
```suggestion

```

Reply:
Fixed.